test/fixtures/v1-80517-estes-park.json

```diff
@@ -68,6 +71,32 @@
       "ami_qualification": null,
       "eligible": true,
       "short_description": "Up to $900 off a heat pump water heater UEF 3.3. ENERGY STAR certified with CTA-2045 certified communication control."
+    },
+    {
+      "payment_methods": [
+        "tax_credit"
+      ],
+      "authority_type": "state",
```

Comment:
It is a little weird to me that some of these incentives are now presenting themselves when I assume they should have been included in a previous PR.  

I just touched this file and CO incentives a few days ago. https://github.com/rewiringamerica/api.rewiringamerica.org/pull/463/files

So, was this being excluded silently? I checked the spreadsheet and it looks like I was also the last to touch it. I'm presuming this was already present when I ran the spreadsheet to json script last.

Reply:
The item in the spreadsheet (and JSON) was `other` before, I think because it wasn't clear how to represent heat pumps and water heaters as a single `item` value. Now that I've updated it to be non-`other` values, it gets served from the API.